objects/rct2ww/ride/rct2.ww.coffeecu.json

```
@@ -413,7 +413,7 @@
             "pt-BR": "Xícaras Giratórias"
         },
         "description": {
-            "en-GB": "Riders ride in pairs of seats rotating around the ends of three long rotating arms",
+            "en-GB": "Riders ride in pairs of themed seats rotating around a large animated coffee mill",
```

From what I gather, coffee grinder (rather than coffee mill) appears to be most used translation of Kaffeemühle.

```suggestion
            "en-GB": "Riders ride in pairs of themed seats rotating around a large animated coffee grinder",
```